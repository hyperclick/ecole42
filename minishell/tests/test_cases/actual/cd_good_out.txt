/Users/darugula/git/e58/minishell/e59/minishell/tests
/Users/darugula/git/e58/minishell/e59/minishell/tests/test_cases/actual
/
cd /Users/darugula
/Users/darugula
cd -
/
cd /Users/darugula/git
/Users/darugula/git
cd cd
/Users/darugula/git
cd ..
/Users/darugula
cd /////
/
/
/
/bin
