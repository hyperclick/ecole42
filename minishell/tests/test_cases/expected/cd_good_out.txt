/mnt/c/git/e59/minishell/tests
/mnt/c/git/e59/minishell/tests/test_cases/actual
/
cd /home/darugula
/home/darugula
cd -
/
cd /home/darugula/git
/home/darugula/git
cd cd
/home/darugula/git
cd ..
/home/darugula
cd /////
/
/
/
/bin
